warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -66,9 +66,27 @@ def open_browser(self, browser_name='firefox', webdriver_remote_url=False,
         browser_name = browser_name
         browser = self._make_browser(browser_name, desired_capabilities,
                                      profile_dir, webdriver_remote_url,
-                                     binary=binary, gecko_path=gecko_path)
+                                    binary=binary, gecko_path=gecko_path)
+        print_info("The Selenium Webdriver version is '{0}'".format(webdriver.__version__))
+        if browser:
+            browser_detail_dict = self.get_browser_details(browser)
+            browser_version = browser_detail_dict['version']
+            print_info("The Browser '{0}' version is '{1}'".format(browser_name, browser_version))
```

Comment:
Use loop to print the details of the dictionary.

Reply:
Good idea and comment. Done!